--- arXiv.xsd	2014-06-17 12:42:29.000000000 -0700
+++ arXiv.xsd.patched	2014-06-17 12:42:21.000000000 -0700
@@ -38,6 +38,7 @@
    <element name="doi"  minOccurs="0" maxOccurs="1" type="string"/>
    <element name="proxy"  minOccurs="0" maxOccurs="1" type="string"/>
    <element name="license"  minOccurs="0" maxOccurs="1" type="string"/>
+   <element name="acm-class"  minOccurs="0" maxOccurs="1" type="string"/>
   </choice>
  </complexType>
 
