--- arXivRaw.xsd	2014-02-18 07:07:42.000000000 -0800
+++ arXivRaw.xsd.patched	2014-06-17 13:26:00.000000000 -0700
@@ -31,6 +31,7 @@
   <element name="doi"  minOccurs="0" maxOccurs="1" type="string"/>
   <element name="license"  minOccurs="0" maxOccurs="1" type="anyURI"/>
   <element name="abstract"  minOccurs="0" maxOccurs="1" type="string"/>
+  <element name="proxy"  minOccurs="0" maxOccurs="1" type="string"/>
  </sequence>
 </complexType>
 
